### PR TITLE
Feature/tr 5440 dispatch trace events

### PR DESCRIPTION
**Pull request by btamas**
Related to https://oat-sa.atlassian.net/browse/TR-5440

- Add `play`, `stop`, `record` and `reset` events.

Test:
- Create an item with that PCI, publish it and check logged events.
- Check migration script with: `php index.php 'oat\tao\scripts\tools\Migrations' -c migrate -v 'oat\qtiItemPci\migrations\Version202308031344181465_qtiItemPci'`
- Play with it in playground. Run `npx grunt connect:dev:keepalive qtiitempcitest` from `/tao/views/build`

### CODE REVIEW

File: views/js/pciCreator/ims/audioRecordingInteraction/runtime/audioRecordingInteraction.js [hunk 1]
```diff
@@ -1038,9 +1074,17 @@ define([
          * @param {Object} [state] - the json serialized state object, returned by previous call to getStatus(), use to initialize an
          */
         getInstance: function getInstance(dom, config, state) {
+            const dispatchInteractiontraceEvent = detail => {
+                const interactiontraceEvent = new CustomEvent('interactiontrace', {
```
Comment (jsconan) [inline views/js/pciCreator/ims/audioRecordingInteraction/runtime/audioRecordingInteraction.js:1078]: **question:** Is the event name the expected one? In the ticket, I rather see `feedtrace`.

**Review by jsconan — APPROVED** [review]
It looks good.
I only have one non-blocking question.

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code respects code style rules
- [x] New code respects best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic

File: views/js/pciCreator/ims/audioRecordingInteraction/runtime/audioRecordingInteraction.js (hunk 1 above)
Comment (btamas) [reply views/js/pciCreator/ims/audioRecordingInteraction/runtime/audioRecordingInteraction.js:1081]: As we discussed, it is not an issue. I've added a comment to the ticket.

**Review by marpesia — APPROVED** [review]
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code respects code style rules
- [ ] New code is respecting best practices
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful
- [ ] Pull request's target is not `master`

**Pull request merged**